src/Core/View.js

```diff
@@ -670,35 +677,65 @@ class View extends THREE.EventDispatcher {
     }
 
     /**
-     * Returns features under the mouse, for a set of specified layers.
+     * Searches for {@link Feature} in {@link ColorLayer}, under the mouse of at
+     * a specified coordinates, in this view.
      *
-     * @param {MouseEvent|Object} mouseOrEvt - A MouseEvent, or a screen
-     * coordinates.
-     * @param {number} [radius=3] - The precision of the picking, in pixels.
-     * @param {Layer[]} [where] - The layers to look into. If not specified, all
-     * `GeometryLayer` and `ColorLayer` layers of this view will be looked in.
+     * @param {Object} mouseOrEvt - Mouse position in window coordinates (from
+     * the top left corner of the window) or `MouseEvent` or `TouchEvent`.
+     * @param {number} [radius=3] - The picking will happen in a circle centered
+     * on mouseOrEvt. This is the radius of this circle, in pixels.
+     * @param {...ColorLayer|GeometryLayer|string} [where] - The layers to look
+     * into. If not specified, all {@link ColorLayer} and {@link GeometryLayer}
+     * layers of this view will be looked in.
      *
      * @return {Object} - An object, with a property per layer. For example,
      * looking for features on layers `wfsBuilding` and `wfsRoads` will give an
      * object like `{ wfsBuilding: [...], wfsRoads: [] }`. Each property is made
      * of an array, that can be empty or filled with found features.
      *
      * @example
-     * view.pickFeaturesAt({ x, y });
-     * view.pickFeaturesAt({ x, y }, 1, ['wfsBuilding']);
-     * view.pickFeaturesAt({ x, y }, 3, ['wfsBuilding', myLayer]);
+     * view.pickFeaturesOnColorLayersAt({ x, y });
```

Comment:
`pickFeaturesOnColorLayersAt` => update the documentation.

Reply:
Yes, it was an error, thanks !